tests/tests_nbde_server_rotate_keys.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Expected behavior of nbde_server_rotate_keys is the following:
+# - if there are no keys, create new keys
+# - if there are keys, do not touch them, unless set
+# - if set, rotate existing keys, if any, plus create new keys
+- name: Verify behavior of nbde_server_rotate_keys
+  hosts: all
+
+  tasks:
+    - name: Ensure we have keys
+      import_role:
+        name: nbde_server
+
+    - name: Gather keys
+      find:
+        paths: "{{ __nbde_server_keydir }}"
+        file_type: file
+        recurse: no
+        patterns:
+          - '*.jwk'
+          - '.*.jwk'
+      check_mode: yes
+      register: existing_keys
+
+    - name: Run with nbde_server_rotate_keys not set to check keys
+      import_role:
+        name: nbde_server
+      vars:
+        nbde_server_rotate_keys: false
+
+    - name: Gather keys after running with nbde_server_rotate_keys not set
+      find:
+        paths: "{{ __nbde_server_keydir }}"
+        file_type: file
+        recurse: no
+        patterns:
+          - '*.jwk'
+          - '.*.jwk'
+      check_mode: yes
+      register: existing_keys_rotate_not_set
+
+    - name: Check whether keys changed - nbde_server_rotate_keys not set
+      assert:
+        that: "{{ existing_keys.files == existing_keys_rotate_not_set.files }}"
```

Comment:
Is it necessary to have the test enclosed in `"{{ .... }}"`?  If it is not needed, I would prefer to drop it.
I know there are other places in the code where there are `when` and `that` tests that use `"` - don't worry about those places which are not directly related to this PR, but please change this and other `that` tests to not enclose the tests in `"{{ .... }}"` if they are not needed.

Reply:
I tested locally and it seems we don't need those in these cases, so I will be removing them.